src/OSPSuite.UI/Views/Importer/ImporterDataView.cs

```diff
@@ -34,6 +34,19 @@ public ImporterDataView()
          dataViewingGridView.OptionsBehavior.Editable = false;
       }
 
+      public override void InitializeResources()
+      {
+         base.InitializeResources();
+         //TODO move to constant file
+         Caption = "Source";
+         btnImport.InitWithImage(ApplicationIcons.Import, "Load current sheet");
```

Comment:
this is how we define buttons so that they have a size consistent throughout the app + an icon